src/views/blocks/tickets/footer.php

```diff
@@ -20,8 +20,16 @@
 $tickets         = $this->get( 'tickets' );
 $currency_symbol = $this->get( 'currency_symbol' );
 $provider        = $this->get( 'provider' );
-$cart_url        = $provider->get_cart_url();
-$checkout_url    = $provider->get_checkout_url();
+if ( method_exists( $provider, "get_cart_url" ) ) {
+	$cart_url = $provider->get_checkout_url();
+} else {
+	$cart_url = '';
+}
+if ( method_exists( $provider, "get_cart_provider" ) ) {
```

Comment:
:no_entry_sign: String "get_cart_provider" does not require double quotes; use single quotes instead

<sub>hash: 6bc7192ee9ce5897a4cb585d932591b7</sub>

Reply:
```suggestion
if ( method_exists( $provider, 'get_cart_provider' ) ) {
```